src/App.js

```diff
@@ -96,9 +95,6 @@ class App extends React.Component {
           }
         } else if ("0123456789)".includes(result[result.length - 2])) {
           if ("/*+-".includes(result[result.length - 1])) {
-            if ("(0123456789".includes(button)) {
-              result = result + button;
-            }
             if (".".includes(button)) {
```

Comment:
Both of these conditions are already covered by the code on line 48 to 51 in the onClick function.